Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.cxx

```diff
@@ -2597,6 +2609,27 @@ void vtkMRMLMarkupsNode::AddMeasurement(vtkMRMLMeasurement* measurement)
   this->Measurements->AddItem(measurement);
 }
 
+//---------------------------------------------------------------------------
+void vtkMRMLMarkupsNode::SetMeasurementAsDefault(int id)
```

Comment:
Not neede anymore since we changed the design (as you suggested in the other comments) and the "primary" measurement is not anymore needed (i.e. I don't display anymore in the tree the Description field)